TUTORIAL.md

```diff
@@ -688,6 +688,14 @@ the CRUD paradigm and have just `create`, `delete`, and `update` mutations.
 While this is a decent starting place, it is insufficient for a proper GraphQL
 API.
 
+As a starting point, do not default to the CRUD verb names. Database
```

Comment:
Reading out in order, I this this would fit better after the next section? We want to talk about determining the logical actions first, and _then_ move onto naming.

We could give this a heading too: `### Naming the mutations`

Reply:
That makes a lot of sense, I've added a new commit